app/test_app/functional/api/test_user_api.py

```diff
@@ -235,10 +251,11 @@ def get_transfer_account_ids(users):
     # User 3 is in Org 2
     # User 4 is in Org 2
     # User 5 is in Org 2
+    # User 6 is in Org 2
     response = get_user_endpoint('1,2')
     assert response.status_code == 200
     users_list = response.json['data']['users']
-    assert get_transfer_account_ids(users_list) == [5, 4, 3, 1]
+    assert get_transfer_account_ids(users_list) == [6, 5, 4, 3, 1]
```

Comment:
This is a side effect of `create_transfer_account_user_2` object in `create_credit_transfer`